vhdbuilder/packer/test/run-test.sh

```diff
@@ -6,6 +6,13 @@ TEST_RESOURCE_PREFIX="vhd-test"
 TEST_VM_ADMIN_USERNAME="azureuser"
 TEST_VM_ADMIN_PASSWORD="TestVM@$(date +%s)"
 
+if [ "$OS_TYPE" == "Linux" ]; then
```

Comment:
This is a workaround, going to extract the code used for Windows and Linux so that we don't too much dependency